sql/logic_test.go

```diff
@@ -376,10 +376,14 @@ func (t *logicTest) execStatement(stmt logicStatement) {
 	switch {
 	case stmt.expectErr == "":
 		if err != nil {
-			t.Fatalf("%s: expected success, but found %v", stmt.pos, err)
+			t.Fatalf("%s: expected success, but found %q", stmt.pos, err)
```

Comment:
In this case I've copied it from below where error was compared to a user
supplied string. I think using %q is valid for that case and I've changed
this for consistency.

On Thu, Aug 6, 2015 at 11:26 AM Tobias Schottdorf <notifications@github.com>
wrote:

> In sql/logic_test.go
> <https://github.com/cockroachdb/cockroach/pull/1994#discussion_r36427693>:
>
> > @@ -376,10 +376,14 @@ func (t *logicTest) execStatement(stmt logicStatement) {
> >  	switch {
> >  	case stmt.expectErr == "":
> >  		if err != nil {
> > -			t.Fatalf("%s: expected success, but found %v", stmt.pos, err)
> > +			t.Fatalf("%s: expected success, but found %q", stmt.pos, err)
>
> printing error as %q is new to me. Do you have an explanation?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1994/files#r36427693>.
>